### DISCUSSION TITLE
How to preview changes in docs code locally

### DISCUSSION THREAD

**Discussion by germanfrelo** [eslint/eslint#18336] [Q&A] [answered]
I want to contribute to improve the CSS of the eslint.org/docs part, but I don't know how to do it easily. I cloned this repo. Is it possible to make changes to the code of the docs directory locally while previewing them in the browser running a local server of the site? I also found out that I can preview the site locally in the eslint.org repo, but the docs part is not there, only here.

**Comment by mdjermanovic** [comment, marked as answer]
> Is it possible to make changes to the code of the docs directory locally while previewing them in the browser running a local server of the site?

Yes, you can run `npm start` in the docs directory.

https://github.com/eslint/eslint/blob/9048e2184c19799bb9b8a5908345d4ce05020c41/docs/package.json#L23

You'd also need to install docs dependencies.

```sh
$ cd docs
$ npm i
$ npm start
```

Here are more detailed instructions: https://github.com/eslint/eslint/blob/main/docs/README.md

**Reply by germanfrelo** [reply]
Thank you! I didn't see that part.
